File: voiceping-sdk/src/main/java/com/smartwalkie/voicepingsdk/listeners/IncomingAudioListener.java
package com.smartwalkie.voicepingsdk.listeners;

import com.smartwalkie.voicepingsdk.models.Message;

/**
 * Created by sirius on 7/11/17.
 */

public interface IncomingAudioListener {

    void onMessageReceived(Message message);

    void onConnectionFailure();
}

File: voiceping-sdk/src/main/java/com/smartwalkie/voicepingsdk/listener/IncomingAudioListener.kt
package com.smartwalkie.voicepingsdk.listener

import com.smartwalkie.voicepingsdk.exception.VoicePingException
import com.smartwalkie.voicepingsdk.model.Message

/**
 * Created by sirius on 7/11/17.
 */
interface IncomingAudioListener {

    fun onMessageReceived(message: Message)

    fun onConnectionFailure(e: VoicePingException)
}